### DISCUSSION TITLE
Is there a way to use it with Google Colab?

### DISCUSSION THREAD

**Discussion by DelsinClade** [Purfview/whisper-standalone-win#369] [Q&A] [answered]
I've really enjoyed this program. However, I've used Google Colab for a long time because of my poor PC’s performance.

So, I was wondering if there’s a way to set up a Google Colab notebook.

I’m mainly using it for --sentence as a parameter and the medium model.

Thank you in advance

**Comment by Purfview** [comment, marked as answer]
Dupe of https://github.com/Purfview/whisper-standalone-win/discussions/318
